modules/webapp/src/main/elm/Messages/Comp/UiSettingsForm.elm

```diff
@@ -182,16 +182,16 @@ Variablen sind verfügbar:
 - `{{concPerson}}` die betreffende Person
 - `{{concEquip}}` die betreffende Aussstattung
 - `{{concerning}}` Person und Ausstattung, getrennt durch ein Komma
-- `{{fileCount}}` die Anzahl von Anhängen des Dokuments
-- `{{dateLong}}` das Dokument-Datum lang formatiert
-- `{{dateShort}}` das Dokument-Datum kurz formatiert (yyyy/mm/dd)
-- `{{dueDateLong}}` das Fälligkeits-Datum lang formatiert
-- `{{dueDateShort}}` das Fälligkeits-Datum kurz formatiert (yyyy/mm/dd)
+- `{{fileCount}}` die Anzahl von Anhängen des Dokumentes
```

Comment:
"Dokuments" is also correct (Duden). In my opinion "Dokuments" sounds a bit better ;)

Reply:
I don't care here :-) What do you think @Lucki ?

Reply:
The key here is probably *sounds*. It *looks* strange to me when written that way.

Reply:
OK. Since both variants are correct and the spelling is a matter of taste, I withdraw my objection.